src/components/learner-credit-management/SpendTableEnrollmentDetails.jsx

```diff
@@ -29,6 +32,15 @@ const SpendTableEnrollmentDetailsContents = ({
         <Hyperlink
           className="x-small"
           destination={`${getConfig().ENTERPRISE_LEARNER_PORTAL_URL}/${enterpriseSlug}/course/${row.original.courseKey}`}
+          onClick={() => {
+            sendEnterpriseTrackEvent(
+              enterpriseId,
+              EVENT_NAMES.LEARNER_CREDIT_MANAGEMENT.BUDGET_DETAILS_SPENT_DATATABLE_VIEW_COURSE,
+              {
+                courseUUID: row.original.uuid,
```

Comment:
Similar to the other comment, worth verifying that this isn't referring to the transaction/redemption UUID (when I believe the intent is to include course identifier).

Reply:
resolved by referencing the courseKey 👍🏽